### PR TITLE
Package release 0.8.1

### PR DESCRIPTION
**Pull request by rram**
:eyeglasses: @spladug @kjoconnor

### CODE REVIEW

File: debian/patches/drop-azure-tests.patch [hunk 1]
```diff
@@ -0,0 +1,308 @@
+Description: Remove Azure tests.
```
Comment (spladug) [inline debian/patches/drop-azure-tests.patch]: Realized after the fact that you might be able to just skip these tests in the `rules` file

```
override_dh_auto_test:
    PYTHONPATH=. py.test --ignore=tests/test_wabs_deleter.py --ignore=tests/test_wabs...
```

https://pytest.org/latest/example/pythoncollection.html

**Comment by spladug** [comment]
:nail_care:

**Comment by rram** [comment]
:haircut: @spladug

**Comment by spladug** [comment]
:fish:

**Comment by rram** [comment]
c89b4546e0fb7cffa7ac39e45dae2cf76cf1f10d

**Pull request closed without merging**